index.js

```diff
@@ -133,7 +133,7 @@ module.exports = {
       }
     ],
     'no-use-before-define': 0,
-    'callback-return': 2,
+    'callback-return': 0,
```

Comment:
Mixed feelings about this one

I feel like it's more typical in nodejs where you usually have a lot of callbacks

https://eslint.org/docs/rules/callback-return#enforce-return-after-callback-callback-return

Reply:
The main reason I don't like this rule is for callbacks that are passed as props:
```js
const callback = () => {
  if (props.callOnChange) {
    return props.onChange();
  }
};

return (
  <Example callback={ callback } />
);
```

In this example, I don't want to expose the return value of `props.onChange` to the `Example` component, but the linting rule forces me to do so without adding any real value.

Reply:
Also it's a bit of a dubious rule, because I can get around it just by renaming `callback` to `notCallback`. Like you said, this rule probably makes more sense in a node environment.